src/bitwarden_exporter/bw_list_process.py

```
@@ -0,0 +1,277 @@
+"""
+Process bitwarden items.
+"""
+
+import json
+import logging
+import os.path
+import shutil
+from datetime import datetime, timezone
+from typing import Any, Dict, List, Optional
```

Ruff (0.14.1)

`typing.Dict` is deprecated, use `dict` instead

(UP035)